qualification/Jenkinsfile

```diff
@@ -171,7 +174,7 @@ pipeline {
         <br>
         <i>Note: This is an Automated email notification.</i>""",
         recipientProviders: [developers(), requestor()],
-        subject: 'Qualification Tests - katgpucbf - $BUILD_STATUS!',
+        subject: "${params.environment} Qualification Tests - katgpucbf - \$BUILD_STATUS!",
```

Comment:
Why do you need to add the backslash before `$BUILD_STATUS!` ?

Reply:
I'll also admit, I was expecting something more like 
* `"Qualification Tests - ${params.environment} - katgpucbf - \$BUILD_STATUS!"`
* i.e. The environment was another identifier in the "hierarchy"

I will concede that the whole point of this was to decipher Lab/Karoo as fast as possible
* Not to make it as aesthetically pleasing as possible

Reply:
> Why do you need to add the backslash before $BUILD_STATUS! ?

It's to ensure the right thing interprets it. Without the backslash, Groovy tries to interpolate the variable when it reads the Jenkinsfile - but BUILD_STATUS is not set as a Groovy variable (it can't be, since the job hasn't run yet). With the backslash, Groovy treats it as a literal `$`, but the emailext plugin substitutes the build status when it formulates the email.

Reply:
> I'll also admit, I was expecting something more like
>
>    "Qualification Tests - ${params.environment} - katgpucbf - \$BUILD_STATUS!"
>    i.e. The environment was another identifier in the "hierarchy"

It's not a hill I'd die on, but I was trying to make it somewhat like the description on the Jenkins page:

![image](https://github.com/user-attachments/assets/1261b23d-1b07-40b9-9436-28904926ae13)